apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -63,10 +67,38 @@ interface WizardStateInterface {
     [ key: string ]: any;
 }
 
+/**
```

Comment:
move these interfaces to the relevant file in the following models folder
`https://github.com/wso2/identity-apps/tree/master/apps/console/src/features/users/models`